oscar_schedules/schedule.py

```diff
@@ -82,7 +82,7 @@ def compute_overlap_periods(self, lower, upper):
 
         return schedules
 
-    def create_default_schedule():
+    def create_default_schedule(self):
```

Comment:
@pvernier 
create_default_schedule is a static factory method, it should not have self

Reply:
can you update the pull? I will then merge it

Reply:
Done